mlflow/tracing/trace_manager.py

```diff
@@ -109,14 +109,14 @@ def start_detached_span(
             if not request_id:
                 request_id = self.get_or_create_request_id(otel_span.get_span_context().trace_id)
 
-            span = MlflowSpanWrapper(otel_span, request_id=request_id, span_type=span_type)
+            span = LiveSpan(otel_span, request_id=request_id, span_type=span_type)
             self.add_or_update_span(span)
             return span
         except Exception as e:
             _logger.warning(f"Failed to start span {name}: {e}")
```

Comment:
@B-Step62 Shouldn't we throw if a span can't be created? Else, it's hard for users to recover from this error (e.g. try / catch) since there's no exception to condition on.